modal_proto/api.proto

```diff
@@ -1776,6 +1788,15 @@ message SandboxGetTaskIdResponse {
   string task_id = 1;
 }
 
+message SandboxGetTunnelsRequest {
+  bool poll = 1;
+  string sandbox_id = 2;
+}
+
+message SandboxGetTunnelsResponse {
+  repeated TunnelData tunnels = 1;
```

Comment:
With this response how do you indicate "no tunnels, stop polling" vs "no tunnels _yet_"

Reply:
#2171